docs/pages/build-on-cyber/contract-deployment.md

```diff
@@ -54,7 +54,7 @@ Deploying a contract with Forge is a simple process that can be done with a sing
 To deploy the Counter.sol contract, use the command that corresponds to the Cyber chain's RPC URL while running the forge create command:
 
 ```bash
-    forge create --rpc-url "RPC URL" //Insert your RPC URL here
+    forge create --rpc-url "https://cyber-testnet.alt.technology/" \ 
```

Comment:
I made the updates in this file too. @akasuv tagging for visibility to avoid conflicts.